src/components/Calculator.css

```diff
@@ -0,0 +1,25 @@
+.grid-container {
+  width: 40%;
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  gap: 1px;
+  background-color: #cdd2d6;
+  padding: 10px;
+}
+
+.grid-container > button {
+  background-color: rgba(255, 255, 255, 0.8);
+  text-align: center;
+  padding: 20px 0;
+  font-size: 30px;
+  height: 100px;
+}
+
+.item1 {
+  grid-column: 1 / 5;
+  text-align: right !important;
+}
+
+.item18 {
+  grid-column: 1 / 3;
+}
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/158434063-453e9dfc-c2ae-4a62-a25c-3019a768cf6d.png)

Therefore:

- [ ] Please, kindly match the given design on your project.

Your Implementation | Target design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/158434256-95e389e1-42f8-402b-8467-0a5f2ba84724.png) | ![image](https://user-images.githubusercontent.com/57421823/158434325-bac86723-032f-4c3d-8738-95b13cea447e.png)

Hint:  Focus on the colors 💪